docs/api-reference/io-columns-binary-classification.md

```diff
@@ -1,6 +1,9 @@
 ### Input and Output Columns
 The input label column data must be <xref:System.Boolean>.
-The input features column data must be a known-sized vector of <xref:System.Single>. This trainer outputs the following columns:
+
```

Comment:
Not sure if we need empty lines.

Reply:
we don't need this empty line. it creates a new paragraph, which we don't want.

---
In reply to: [277870070](https://github.com/dotnet/machinelearning/pull/3533#discussion_r277870070) [](ancestors = 277870070)